oasislmf/utils/inputs.py

```diff
@@ -62,7 +81,7 @@ def update_config_keys(self):
     def load_config_file(self):
         try:
             with io.open(self.config_fp, 'r', encoding='utf-8') as f:
-                return json.load(f)
+                return dict((k.lower(), v) for k, v in json.load(f).items())
```

Comment:
If it is ok for you I prefer the new dict formatting, ie:
{k.lower(): v for k,v in json.load(f).items()}